modules/cloudbuild/main.tf

```diff
@@ -15,7 +15,7 @@
  */
 
 locals {
-  cloudbuild_project_id       = var.project_id != "" ? var.project_id : format("%s-%s", var.project_prefix, "cloudbuild")
+  cloudbuild_project_id       = var.project_id != "" ? var.project_id : format("%s-%s-%s", var.project_prefix, "cloudbuild", random_id.suffix.hex)
```

Comment:
Project factory module already adds a random suffix within this module https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/c8db8f3ca09ebb9a99155ac10727f01ec7fd7645/modules/cloudbuild/main.tf#L42

We should construct a `project_id` in example foundations and pass into this module via `project_id` in https://github.com/terraform-google-modules/terraform-example-foundation/blob/34c9125306271994e86a1a63980b7c7f5af01291/0-bootstrap/main.tf#L115 for more randomization

Reply:
Hey @bharathkkb 
Do you mean construct it in the same format as we do here ie. `project_prefix-cloudbuild-randomhex`
example foundation also references it here https://github.com/terraform-google-modules/terraform-example-foundation/blob/34c9125306271994e86a1a63980b7c7f5af01291/0-bootstrap/main.tf#L176